### ISSUE TITLE
FAR and Umbra parachutes don’t play nice

### ISSUE THREAD

**Issue by Mace134** [ferram4/Ferram-Aerospace-Research#132]
I’m not sure whose door to leave this at so I’ll be putting this on both issue trackers and I apologize if this has been brought to your attention before. FAR seems to neuter the stock ModuleParachute, which isn’t a problem for parts that have RealChuteFAR to fill the gap but the Umbra parts don’t seem to like using a different module and thus spawn in editor and on the launch pad with the parachute as a fully deployed object and completely useless. They don’t appear in staging, and although their events can be tied to action groups it doesn’t do anything. This was not my experience in 1.0.5.

Below is what I would think is the pertinent info but if you need anything else please let me know.

Kerbal Space Program - 1.1.0.1230 -- Windows x64
ModuleManager.2.6.22
Firespitter v7.1.5950.29855
Ferram Aerospace Research 0.15.6
Survivability Pack v0.5.1.0
Sounding Rockets v0.4.1.0
*USITools v1.0.0.0 (As reported in KSP.log)
*USITools v0.7.1.0 (As reported in USITools.version)

**Comment by StupidChris** [comment]
As mentioned with RealChuteFAR, this is me you'd want to talk to. Care to explain what the Umbra parachutes are doing that's different from the normal stock chutes?

**Comment by ferram4** [comment]
Uhh... do you have ModularFlightIntegrator 1.1.3 installed?  If not, then it would make sense that everything is broken, because FAR can't even load in that case.

**Comment by Mace134** [comment]
Well some more information, I did have the ModularFlightIntegrator that was in the FAR zip. I thought I had 100% reproducibility as I have been toying with it since last night. I just did a complete fresh install of KSP, FAR, and Umbra and all the parachutes worked perfectly. Then I installed full RealChute, which I normally play with but believed I had eliminated and got the bug to happen again.

StupidChris, the problem appears to be that with FAR and RealChute the parachutes added in some of the Umbra mods do not work. Specifically that they deploy as expected with the stock ModuleParachute but provide no drag and my craft slam full force into the surface.

My most humble apologies, I imagine after a major patch this can be quite a shit show.
